python/parser.js

```diff
@@ -215,6 +220,9 @@
 const returnSyntax = ({value}) => {
     return `return ${value};`
 }
+function handleHooks(hooks) {
+    if ("beforeCaptureScreenshot" in hooks) return '.before_render_screenshot_hook(\"' + `${hooks.beforeCaptureScreenshot}` + '\")'
```

Comment:
Codacy has a fix for the issue: [Expected { after 'if' condition.](https://app.codacy.com/gh/applitools/sdk.coverage.tests/pullRequest?prid=8528488)
```suggestion
    if ("beforeCaptureScreenshot" in hooks) {return '.before_render_screenshot_hook(\"' + `${hooks.beforeCaptureScreenshot}` + '\")'}
```